docs/en/Configuration-Properties.md

```diff
@@ -0,0 +1,112 @@
+
+<table class="table table-striped">
+<tr><th>Property Name</th><th>Default</th><th>Description</th></tr>
+{% for item in site.data.table.master-configuration %}
+  <tr>
+    <td>{{ item.propertyName }}</td>
+    <td>{{ item.defaultValue }}</td>
+    <td>{{ site.data.table.en.master-configuration[item.propertyName] }}</td>
+  </tr>
+{% endfor %}
+</table>
+
+### Worker Configuration
+
+The worker configuration specifies information regarding the worker nodes, such as the address and
+the port number.
+
+<table class="table table-striped">
+<tr><th>Property Name</th><th>Default</th><th>Description</th></tr>
+{% for item in site.data.table.worker-configuration %}
+  <tr>
+    <td>{{ item.propertyName }}</td>
+    <td>{{ item.defaultValue }}</td>
+    <td>{{ site.data.table.en.worker-configuration[item.propertyName] }}</td>
+  </tr>
+{% endfor %}
+</table>
+
+### User Configuration
+
+The user configuration specifies values regarding file system access.
+
+<table class="table table-striped">
+<tr><th>Property Name</th><th>Default</th><th>Description</th></tr>
+{% for item in site.data.table.user-configuration %}
+  <tr>
+    <td>{{ item.propertyName }}</td>
+    <td>{{ item.defaultValue }}</td>
+    <td>{{ site.data.table.en.user-configuration[item.propertyName] }}</td>
+  </tr>
+{% endfor %}
+</table>
+
+### Resource Manager Configuration
+
+When running Alluxio with resource managers like Mesos and YARN, Alluxio has additional configuration options.
+
+<table class="table table-striped">
+<tr><th>Property Name</th><th>Default</th><th>Description</th></tr>
+{% for item in site.data.table.cluster-management %}
+  <tr>
+    <td>{{ item.propertyName }}</td>
+    <td>{{ item.defaultValue }}</td>
+    <td>{{ site.data.table.en.cluster-management[item.propertyName] }}</td>
+  </tr>
+{% endfor %}
+</table>
+
+### Security Configuration
+
+The security configuration specifies information regarding the security features, such as authentication and file permission. Settings for authentication take effect for master, worker, and user. Settings for file permission only take effect for master. See [Security](Security.html) for more information about security features.
+
+<table class="table table-striped">
+<tr><th>Property Name</th><th>Default</th><th>Description</th></tr>
+{% for item in site.data.table.security-configuration %}
+  <tr>
+    <td>{{ item.propertyName }}</td>
+    <td>{{ item.defaultValue }}</td>
+    <td>{{ site.data.table.en.security-configuration[item.propertyName] }}</td>
+  </tr>
+{% endfor %}
+</table>
+
+### Configure Multihomed Networks
+
+Alluxio configuration provides a way to take advantage of multi-homed networks. If you have more than one NICs and you want your Alluxio master to listen on all NICs, you can specify `alluxio.master.bind.host` to be `0.0.0.0`. As a result, Alluxio clients can reach the master node from connecting to any of its NIC. This is also the same case for other settings suffixed with `bind.host`.
```

Comment:
Removed. Since they're already set to 0.0.0.0 by default.

Agree with the network configuration session. However, I am not very familiar with that part. Will you or calvin be able to come up with that session?